app/services/sufia/workflow/workflow_by_admin_set_strategy.rb

```diff
@@ -0,0 +1,15 @@
+module Sufia
+  module Workflow
+    class WorkflowByAdminSetStrategy
+      def initialize(work, attributes)
+        @work = work
```

Comment:
Can I safely remove the work variable in the initialize method? Or do I just not use it? Thanks.